src/main/java/build/buildfarm/common/redis/RedisPriorityQueue.java

```diff
@@ -141,14 +142,15 @@ public boolean removeAll(JedisCluster jedis, String val) {
    * @details This pops the element from one queue atomically into an internal list called the
    *     dequeue. It will wait until the timeout has expired. Null is returned if the timeout has
    *     expired.
-   * @param timeout_s Timeout to wait if there is no item to dequeue. (units: seconds (s))
+   * @param timeout_ms Timeout to wait if there is no item to dequeue. (units: milliseconds (ms))
    * @return The value of the transfered element. null if the thread was interrupted.
    * @note Overloaded.
    * @note Suggested return identifier: val.
    */
   @Override
-  public String dequeue(JedisCluster jedis, int timeout_s) throws InterruptedException {
-    int maxAttempts = (int) (timeout_s / (pollIntervalMillis / 1000.0));
+  public String dequeue(Jedis jedis, int timeout_ms, ExecutorService service)
```

Comment:
this timeout change (seconds->ms) makes me nervous 😬 did we refactor all of the callers? Is `Duration` a safer type?